draft-ietf-6man-slaac-renum-08.xml

```diff
@@ -313,9 +313,13 @@ The following subsections update <xref target="RFC4191"/>, <xref target="RFC4861
 <vspace blankLines="0" />This allows local hosts to learn about stale configuration information in a timelier manner.</t>
 <t>Honor PIOs with small Valid Lifetimes (<xref target="sig-stale-config"/>): 
 <vspace blankLines="0" />This allows hosts to honor PIOs with a Valid Lifetime less than 2 hours, thus resulting in a timelier reaction to flash-renumbering events.</t>
+<t>
+Recommend routers to support prefix-specific link-local addresses as a configurable behavior, which can be enabled in envinronments susceptible to flash renumbering (<xref target="pslla"/>).
+<vspace blankLines="0" />This allows hosts to detect the prefix change much faster in the absense of any explicit renumbering signals.
```

Comment:
Something that shoudl be noted here (besides my other note on what happens when NUD fails), is that:
* It seems a change in  RFC6724 would be needed for this mechanism to work.
* This would help for cases where:
  + routers implement gulla, and,
  + hosts implement both RFC8028 plus the noted change in RFC6724 are implemented on the host side.

Reply:
> Something that shoudl be noted here (besides my other note on what happens when NUD fails), is that:
> 
> * It seems a change in  RFC6724 would be needed for this mechanism to work.
> * This would help for cases where:
>   
>   * routers implement gulla, and,
>   * hosts implement both RFC8028 plus the noted change in RFC6724 are implemented on the host side.

As discussed above, I do not believe any changes are required for 6724 (well, besides making Rule 5.5 a MUST, which the mist recent update does anyway).

8028 is not really required here.

Reply:
This is my reasoning:
1. WHat you probably want is that, when picking a src addr, you don't pick one that would use a router that is not known to be reachable. -- this would be RFC6724 turf.
2. My note on RFC8028 is that, unless you ahve RFC8028 in place, multi-address/multi-router is broken (in practice it will only work if you ahve a single upstream) -- see: https://www.ietf.org/archive/id/draft-gont-v6ops-multi-ipv6-01.txt .